docs/Examples2/Text.example.purs

```diff
@@ -0,0 +1,97 @@
+import Lumi.Styles.Box (_flex)
+import React.Basic (JSX, fragment)
+
+docs :: JSX
+docs =
+  column
+  $$$ [ T.sectionHeader $$$ "Paragraphs"
+      , T.paragraph_ $$$ "Paragraphs are used for laying out chunks of text as concrete, individual and independent elements in a page."
+      , example
+          $ fragment
+          $ [ T.paragraph_ $$$ "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lacinia imperdiet turpis, blandit mollis purus venenatis non. Curabitur tincidunt magna at purus facilisis tempus in ut massa. Donec egestas orci lacus, eu rhoncus ipsum finibus et. Curabitur eu dapibus massa."
+            , T.paragraph_ $ T.subtext $$$ "Sed a arcu nibh. Aliquam rhoncus luctus gravida. Morbi pharetra sollicitudin arcu sit amet euismod. Maecenas blandit nisi ac metus tristique, ut euismod erat rhoncus. Donec semper ut est ac sodales. Sed elementum laoreet laoreet."
+            ]
+
+      , T.sectionHeader $$$ "Headers"
+      , T.paragraph_ $$$ "Headers are block-level elements used for marking/separating sections of a layout or for displaying important pieces of information in a context."
+      , example
+          $ fragment
+          $ [ T.mainHeader $$$ "MainHeader"
+            , T.title $$$ "Title"
+            , T.sectionHeader $$$ "SectionHeader"
+            , T.subsectionHeader $$$ "SubsectionHeader"
+            ]
+
+      , T.paragraph_ $$$ "Text elements such as headers and paragraphs can be composed inside other block level elements such as boxes, columns and rows."
+      , example
+        $ fragment
+        $ [ T.sectionHeader $ T.strong $$$ "A tiny story"
+          , T.subsectionHeader $$$ "Hello!"
+          , T.paragraph
+            $$$ [ T.text
+                  $ T.emphasized
+                  $$$ "How are you? "
+                , T.text
+                  $$$ "Hope you're doing "
+                , T.text
+                  $ T.strong
+                  $$$ "fine. "
+                , T.text
+                  $ T.subtext
+                  $ T.muted
+                  $$$ "Yes, I do."
+                ]
+          , T.paragraph
+            $$$ [ T.text
+                  $$$ "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam congue ligula et odio rutrum, eu imperdiet ante laoreet. Cras mollis faucibus urna, eu luctus ligula condimentum ut. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vivamus et mi mattis, maximus urna id, luctus neque. Sed non lorem molestie nibh suscipit condimentum id quis enim. Nunc tortor elit, posuere eu metus sed, finibus sagittis est. Fusce dapibus lacus vitae augue vulputate, in convallis lectus congue. "
+                , T.text
+                  $ T.strong
+                  $$$ "Hi! "
+                , T.text
+                  $ T.subtext
+                  $$$ "Hey!"
+                ]
+          , row
+            $$$ [ T.paragraph
+                  $ T.emphasized
+                  $ _flex
```

Comment:
`_flex` maybe belongs on Box the way `muted` is tied to Text, but we can look into that in a separate PR

Reply:
Not sure it does. I believe `flex` may be applied to all block-level elements, and paragraph is  a block element. Maybe we should start encoding element types somehow in the library?